ipv8/test/attestation/trustchain/test_block.py

```diff
@@ -14,25 +14,25 @@ class TestBlock(TrustChainBlock):
     Also used in other test files for TrustChain.
     """
 
-    def __init__(self, transaction=None, previous=None, key=None):
+    def __init__(self, transaction=None, previous=None, key=None, block_type='test'):
         crypto = ECCrypto()
         other = crypto.generate_key(u"curve25519").pub().key_to_bin()
 
         transaction = transaction or {'id': 42}
 
         if previous:
             self.key = previous.key
-            TrustChainBlock.__init__(self, (encode(transaction), previous.public_key, previous.sequence_number + 1,
-                                            other, 0, previous.hash, 0, 0))
+            TrustChainBlock.__init__(self, block_type, (encode(transaction), previous.public_key,
+                                                        previous.sequence_number + 1, other, 0, previous.hash, 0, 0, 0))
         else:
             if key:
                 self.key = key
             else:
                 self.key = crypto.generate_key(u"curve25519")
 
-            TrustChainBlock.__init__(self, (
-                encode(transaction), self.key.pub().key_to_bin(), random.randint(50, 100), other, 0,
-                sha256(str(random.randint(0, 100000))).digest(), 0, 0))
+            TrustChainBlock.__init__(self, (block_type,
+                                            encode(transaction), self.key.pub().key_to_bin(), random.randint(50, 100),
+                                            other, 0, sha256(str(random.randint(0, 100000))).digest(), 0, 0, 0))
```

Comment:
Formatting? Do all these spaces really increase readability?

Reply:
I think it increases readability and it follows [the PEP8 conventions](https://www.python.org/dev/peps/pep-0008/).

Reply:
If it fits in 80 lines that's OK.